app/scrapers/generalized.py

```diff
@@ -20,12 +21,14 @@ class Scraper:
     def __init__(self):
         pass
 
-    def get_page(self, query, startIndex=0):
+    def get_page(self, query, startIndex=0, qtype=''):
         """ Fetch the google search results page
         Returns : Results Page
         """
-        payload = {self.queryKey: query, self.startKey: startIndex}
+        payload = {self.queryKey: query, self.startKey: startIndex,
+                   self.qtype: qtype}
         response = requests.get(self.url, headers=self.headers, params=payload)
+        print(response.url)
```

Comment:
Should we do this?

Reply:
Well that was during testing. I forgot to remove it :sweat_smile: 
I don't know about live link. Can you tell me more?

Reply:
@umangahuja1 Deploy it on heroku and share the link here

Reply:
I am not able to deploy. It is asking for card details. Though I have deployed my apps for free earlier.